models/intel/smartlab-action-recognition-0001/README.md

```diff
@@ -0,0 +1,73 @@
+# smartlab-action-recognition-0001 (composite)
+
+## Use Case and High-Level Description
+
+This is an smartlab action recogntion model that recognizes smartlab actions.
+The model uses smartlab dataset to predict actions.
+The model is classfier for 3 class actions.
+
+## Example of the input data
+<!-- ![](./assets/frame0001.jpg) -->
+
+## Example of the output
+
+`openvino`
+
+## Composite model specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the DSI1867                        | TODO               |
+| Source framework                               | PyTorch\*          |
+
+
+## Encoder model specification
+
+The smartlab-action-recognition-encoder-0001 is a Mobilenet-V2 like backbone with convolutional encoder part of the action recognition.
+
+| Metric  | Value |
+| ------- | ----- |
+| GFlops  | 0.611 |
+| MParams | 3.387 |
+
+### Inputs
+
+Image, name: `input_image`, shape: `1, 1, 224, 224` in the `1, C, H, W` format, where:
```

Comment:
are you sure that model has single channel input? In internal repo I see 3 channels
```suggestion
Image, name: `input_image`, shape: `1, 3, 224, 224` in the `N, C, H, W` format, where:
```

Reply:
fixed